### PR TITLE
supporting direct token authentication and retrieval of active auth_t…

### PR DESCRIPTION
**Pull request by swimclan**
…oken via the api

Alejandro, I am an active user of your RH api wrapper.  It has been a pleasure working with this tool so far.  I am starting to take some of my development to the next level and would like to have a way to authenticate directly to RH using their token so I can have multiple accounts on my server sharing this resource without having to store their usernames and passwords.  

This pull request does two main things:

1. Allows for consumers of your wrapper to use a Robinhood api auth token directly with the library as an alternative authorization method in addition to username & password.

2. Includes a 'getter' for retrieving the currently authenticated auth_token from the wrapper's api so the consumer can store it for future authentication (via item #1 above).

I humbly present this merge request for your consideration.

Thanks,
Matt Herron (@swimclan)

### CODE REVIEW

**Comment by aurbano** [comment]
Looks great, thanks for the PR! Will merge as soon as you clarify the token

Thanks :)

**Comment by swimclan** [comment]
@aurbano token cleared out of README.

**Comment by aurbano** [comment]
Released in v0.12.0

File: README.md [hunk 1]
```diff
@@ -56,14 +57,28 @@ $ npm install robinhood --save
...
     password: ''
 };
+```
+
+### Robinhood API Auth Token
+```js
+//A previously authenticated Robinhood API auth token
+
+var credentials = {
+    token: '2d37glsbcg55iqall98wn6wssd78vb34lpo'
```
Comment (swimclan) [inline README.md]: Yes that was a random token.  not real.  Sorry I should have clarified that

**Pull request merged**